__tests__/pages/__snapshots__/curriculum.test.js.snap

```diff
@@ -776,7 +776,7 @@ exports[`Curriculum Page Should render with basic dummy data 1`] = `
           />
           <a
             class="mx-3"
-            href="#"
+            href="/#"
```

Comment:
All the versions are still the same(`the latest version`) right? So why did this snapshot change?

Reply:
Who knows? hahaha

I think that today it felt that having the slash there was better.

Reply:
Maybe now that we have the versions pinned to a single one this will go away.